Code-4-Community/lucys-love-bus-frontend
# Create Update Profile Info/Set Contacts form

## Issue

Closes #70 
Linked backend PR: https://github.com/Code-4-Community/lucys-love-bus-backend/pull/125

## Changes

- Created Update Profile Info page/Set Contacts PF signup flow page (same thing)
- Made various form fragment components
- Created redux files for contacts
- Linked signup PF form to set contacts page
- Refactored code into signup confirmation page which is now its own container, signup routes there if GM, otherwise set contacts when in the PF signup flow.

## Screenshots

![image](https://user-images.githubusercontent.com/23691775/113461466-b1d3a500-93ea-11eb-843d-f5a0f5b12bb7.png)


## Verification Steps

- Looked in pgadmin to see if children and contacts were properly created.
- Children and contacts were created with correct S3 image links.
- Was able to update contacts multiple times, including photos which updated properly.

## Notes

Backend PR https://github.com/Code-4-Community/lucys-love-bus-backend/pull/125 is required for this ticket to work in production
